.github/workflows/code.yml

```diff
@@ -122,11 +123,18 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
Am I right in thinking you need this so that you can get the latest tag info from the full history?